src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<configuration debug='true'>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">

```

Comment:
we should sanity check that stderr gets captured also.

Reply:
pretty sure it does .. check the jenkins log or run mvn locally

Reply:
OOP ! .. good catch ;)
I'll add it